gpuarray/src/private_config.h

```diff
@@ -0,0 +1,53 @@
+/******************************************************************
```

Comment:
Should I remove them wherever they are not needed? I will do it with a substitution script, it will be fast.